gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/model/Subscription.java

```diff
@@ -361,6 +361,22 @@ public void setFailureCause(String failureCause) {
         this.failureCause = failureCause;
     }
 
+    /**
+     * Consumer can start subscription only if its consumer status is {@link this#consumerStatus} is {@link ConsumerStatus#STOPPED} or {@link ConsumerStatus#STARTED}
+     * @return true when consumer can start the subscription
+     */
+    public boolean canConsumerStart() {
```

Comment:
I would have gone with something like `canBeStartedByConsumer` and ```canBeStoppedByConsumer``` but that may be a matter of taste